Frontend/CO.CDP.OrganisationApp/Pages/Organisation/OrganisationOverview.cshtml

```diff
@@ -147,6 +151,48 @@
                             </authorize>
                         </div>
                     }
+
+                    @if (Model.BuyerInformation != null)
+                    {
+                        <div class="govuk-summary-list__row">
+                            <dt class="govuk-summary-list__key">
+                                @StaticTextResource.Organisation_BuyerType
+                            </dt>
+                            <dd class="govuk-summary-list__value">
+                                <p class="govuk-body">
+                                    @(BuyerOrganisationTypeModel.BuyerTypes[Model.BuyerInformation.BuyerType!])
+                                </p>
+                            </dd>
+                        </div>
+
+                        <div class="govuk-summary-list__row">
+                            <dt class="govuk-summary-list__key">
+                                @StaticTextResource.Organisation_DevolvedRegulations
+                            </dt>
+                            <dd class="govuk-summary-list__value">
+                                <p class="govuk-body">
+                                    @(Model.BuyerInformation.DevolvedRegulations.Count > 0 ? "Yes" : "No")
```

Comment:
We have `StaticTextResource.Global_Yes` and `StaticTextResource.Global_No` we can use here

Reply:
Ah man I missed these. Sorry.

Reply:
Done